address-poisoning-py/src/rules.py

```diff
@@ -35,17 +35,17 @@ def have_addresses_been_detected(transaction_event, zero_value_contracts, low_va
 
 
     @staticmethod
-    def are_all_logs_stablecoins(logs, chain_id):
-        stablecoin_count = 0
+    def are_all_logs_stablecoins_or_base_coins(logs, chain_id):
+        stablecoin_or_base_count = 0
 
         if len(logs) == 0:
             return 0
 
         for log in logs:
-            if str.lower(log['address']) in STABLECOIN_CONTRACTS[chain_id]:
-                stablecoin_count += 1
+            if str.lower(log['address']) in STABLECOIN_CONTRACTS[chain_id] or log['address'] in BASE_TOKENS:
```

Comment:
```suggestion
            if log['address'].lower() in STABLECOIN_CONTRACTS[chain_id] or log['address'].lower() in BASE_TOKENS:
```
address for the `BASE_TOKENS` check should also be lower case. However, just suggesting using `<string>.lower()` versus `str.lower(<string>)`.